apps/docs/content/guides/platform/compute-and-disk-simple.mdx

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
Where is this page intended to go? I couldn't find where it's linked from

Reply:
Ah sorry, this was an alternative version of the compute and disk page that I wanted to preview and get feedback on. It's not linked from anywhere.